source/_integrations/dwd_weather_warnings.markdown

```diff
@@ -1,85 +1,49 @@
 ha_domain: dwd_weather_warnings
 ha_codeowners:
   - '@runningman84'
   - '@stephan192'
   - '@Hummel95'
+  - '@andarotajo'
 ha_platforms:
   - sensor
 ha_integration_type: integration
 ---
 
-The `dwd_weather_warnings` sensor platform uses the [Deutsche Wetter Dienst (DWD)](https://www.dwd.de) as a source for current and advance warnings.
+The `dwd_weather_warnings` integration uses the [Deutscher Wetterdienst (DWD)](https://www.dwd.de) as a source for current and advance weather warnings. The configured sensor checks for data every 15 minutes.
 
-- A name is optional but if multiple regions are used a name will be required.
-- The sensor checks for new data every 15 minutes.
+{% include integrations/config_flow.md %}
 
-## Configuration
-
-To add the DWD WarnApp sensor to your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: dwd_weather_warnings
-    region_name: Hansestadt Hamburg
-```
-
-<div class="note">
-
-- The `region_name` can either be a so called `warncell id` (integer) or a `warncell name` (string). It is heavily advised to use `warncell id` because `warncell name` is not unique in some cases.  
-A list of valid warncell ids and names can be found at [here](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.html).
-- Some of the warncells are outdated but still listed. If setup fails search the list for a similar sounding warncell.
-- If you selected a `warncell name` and the name is not unique `" (not unique used ID)!"` will be added to the reported `region_name`.
-
-</div>
-
-{% configuration %}
-region_name:
-  required: true
-  description: The region name = warncell name (string) or region id = warncell id (integer) taken from DWD homepage.
-  type: [string, integer]
-name:
-  required: false
-  description: The name you would like to give to the warnapp sensor.
-  type: string
-  default: DWD-Weather-Warnings
-monitored_conditions:
-  description: List of warnings you want to be informed about.
-  required: false
-  default: all
-  type: list
-  keys:
-    current_warning_level:
-      description: The current warning level.
-    advance_warning_level:
-      description: The expected warning level.
-{% endconfiguration %}
+{% configuration_basic %}
+Warncell ID or name:
+  description: Identifier of the region. It can either be a so called `warncell ID` (integer) or a `warncell name` (string). It is heavily advised to use `warncell ID` because `warncell name` is not unique in some cases. A list of valid warncell IDs and names can be found [here](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.html). Some of the warncells are outdated but still listed. If the setup fails, search the list for a similar sounding warncell. In case the `warncell name` is not unique, `" (not unique used ID)!"` will be added to the reported `region_name`. Setting this field is required.
```

Comment:
```suggestion
  description: Identifier of the region. It can be a warncell ID (integer) or a warncell name. It is heavily advised to use warncell ID because a warncell name is sometimes not unique. A list of valid warncell IDs and names can be found [here](https://www.dwd.de/DE/leistungen/opendata/help/warnungen/cap_warncellids_csv.html). Some of the warncells are outdated but still listed. If the setup fails, search the list for a similar sounding warncell. If the warncell name is not unique, `" (not unique used ID)!"` will be added to the reported `region_name`. Setting this field is required.
```